arm-sql/2014-04-01/examples/DatabaseSecurityAlertCreateMin.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "securityalert-4799",
+    "serverName": "securityalert-6440",
+    "databaseName": "testdb",
+    "securityAlertPolicyName": "Default",
+    "api-version": "2014-04-01",
+    "databaseSecurityAlertPolicy": {
+      "properties": {
+        "state": "Enabled",
+        "storageAccountAccessKey": "sdlfkjabc+sdlfkjsdlkfsjdfLDKFTERLKFDFKLjsdfksjdflsdkfD2342309432849328476458/3RSD==",
+        "storageEndpoint": "https://mystorage.blob.core.windows.net"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/securityalert-4799/providers/Microsoft.Sql/servers/securityalert-6440/databases/testdb",
+        "name": "Default",
+        "type": "Microsoft.Sql/servers/databases/securityAlertPolicies",
+        "kind": "V12",
+          "properties": {
+            "state": "Enabled",
+            "emailAccountAdmins": "Enabled",
+            "emailAddresses": "",
+            "disabledAlerts": "",
```

Comment:
Is it OK to send an empty string back in the response if the property was not set in the request body OR due to any other reason. An optional property not being present if not set or it being present always with an empty string are two different things. Looks like the latter is true. If that is indeed the case then should such properties have `"default": ""`.
I have seen this for "storageAccountAccessKey" and "storageEndpoint" as well.